.sites.config.example.yml

```diff
@@ -2,6 +2,7 @@
 default:
   database_s3_bucket: S3_BUCKET_NAME
   database_s3_key_prefix_string: prod-db-prefix
+  drupal_admin_uid: 108
```

Comment:
FWIW I liked the old variable name haha. This doesn't have to be an admin user. 🤷🏻‍♂️

Reply:
Updated again!